content/docs/learning-guide/tutorials/environments/create-terrain-from-images.md

```diff
@@ -0,0 +1,314 @@
+
+{{< image-width src="/images/learning-guide/tutorials/environments/macro-material-blending.png" width="800" alt="An example of the difference in variation created by macro material blending." >}}
+
+{{< tip >}}
+In the **Terrain World Renderer** level component, there is a property group named **Detail material configuration** that you can use to configure how far from the camera to stop blending detail materials into the macro material, and the distance over which to fade out the blended detail materials.
+{{< /tip >}}
+
+The following example image demonstrates various material assignments and how they interact on a terrain surface:
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-materials-example.png" width="800" alt="An example of macro and detail materials applied to a terrain." >}}
+
+In this example, there are four materials:
+
+* Magenta - This is the macro material. It's displayed on terrain that is 512 meters from the camera. The blend distance between this macro material and the detail materials is 64 meters.
+
+* Green - This is the default detail material. Detail materials are layered, and the default material is the base layer. The default material is displayed on areas that aren't assigned a detail material through a surface tag name, but are close to the camera.
+
+* Red - This is a detail material. It's applied to the terrain through a surface tag name. The surface tag name is generated by a tiled image gradient that contains a white triangle on a black background.
+
+* Blue - This is a detail material. It's applied to the terrain through a surface tag name. The surface tag name is generated by a tiled image gradient that contains a white circle on a black background.
+
+{{< note >}}
+Notice that the red and blue detail materials blend to purple in areas where the triangles and circles overlap. The is because the image gradients are generating surface tags for both materials in these overlap areas.
+{{< /note >}}
+
+### Apply a macro material
+
+1. With the Terrain Spawner entity selected, add a **Terrain Macro Material** component.
+
+1. In the **Terrain Macro Material** component, supply a color texture and a normal texture.
+
+### Apply detail materials
+
+This section assumes you have created the materials you want to apply to your terrain in Material Editor, and that you have created image gradients that define the detail material areas. To apply the materials to the terrain as detail materials, do the following:
+
+#### Create a surface tag name list
+
+Surface tag names are strings that you can use to refer to parts of the terrain that are associated with specific gradients.
```

Comment:
I'm glad this definition is in here. I wasn't able to understand what surface tags were by looking at the Surface Data Gem doc or Surface Tag Emitter component docs. This is just a callout that it may be helpful to have conceptual docs on what surface tags are. 

```suggestion
_Surface tag names_ are strings that you can use to refer to parts of the terrain that are associated with specific gradients.
```